statsbestcustomers.php

```diff
@@ -168,9 +168,10 @@ public function getData()
 				AND o.valid
 			), 0) as totalValidOrders
 		FROM `'._DB_PREFIX_.'customer` c
+        INNER JOIN `'._DB_PREFIX_.'orders` o ON o.id_customer = c.id_customer 
```

Comment:
This file is really old, full of tabs :joy:

Reply:
Could you use tab everywhere please?

Reply:
@PierreRambaud it's done everywhere

Reply:
@PierreRambaud tabs ? Don't we use spaces everywhere now ? 🤦

Reply:
This file is broken, have tabs and spaces :joy: 
So you're right @NeOMakinG might as weel use spaces :sweat_smile: